src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,28 @@
 package mate.academy;
 
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArrayList;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private List<EventListener> listeners = new CopyOnWriteArrayList<>();
```

Comment:
Consider providing a mechanism to prevent the same listener from being registered multiple times as this could lead to unexpected behavior when events are notified.

Reply:
Agree, you can use Set for example